examples/oemof_0.2/basic_example/basic_example.py

```diff
@@ -0,0 +1,213 @@
+
+                input/output  bgas     bel
+                     |          |        |       |
+                     |          |        |       |
+ wind(FixedSource)   |------------------>|       |
+                     |          |        |       |
+ pv(FixedSource)     |------------------>|       |
+                     |          |        |       |
+ rgas(Commodity)     |--------->|        |       |
+                     |          |        |       |
+ demand(Sink)        |<------------------|       |
+                     |          |        |       |
+                     |          |        |       |
+ pp_gas(Transformer) |<---------|        |       |
+                     |------------------>|       |
+                     |          |        |       |
+ storage(Storage)    |<------------------|       |
+                     |------------------>|       |
+
+Installation requirements:
+---------------------------
+
+This example requires the version v0.2.x of oemof. Install by:
+
+    pip install 'oemof>=0.2,<0.3'
+
+Optional:
+
+    pip install matplotlib
+
+"""
+
+# ****************************************************************************
+# ********** PART 1 - Define and optimise the energy system ******************
+# ****************************************************************************
+
+###############################################################################
+# imports
+###############################################################################
+
+# Default logger of oemof
+from oemof.tools import logger
+from oemof.tools import helpers
+
+import oemof.solph as solph
+from oemof.outputlib import processing, views
+
+import logging
+import os
+import pandas as pd
+import pprint as pp
+
+try:
+    import matplotlib.pyplot as plt
+except ImportError:
+    plt = None
+
+
+solver = 'cbc'  # 'glpk', 'gurobi',....
+debug = False  # Set number_of_timesteps to 3 to get a readable lp-file.
+number_of_time_steps = 24*7*8
+solver_verbose = False  # show/hide solver output
+
+# initiate the logger (see the API docs for more information)
+logger.define_logging(logfile='oemof_example.log',
+                      screen_level=logging.INFO,
+                      file_level=logging.DEBUG)
+
+logging.info('Initialize the energy system')
+date_time_index = pd.date_range('1/1/2012', periods=number_of_time_steps,
+                                freq='H')
+
+energysystem = solph.EnergySystem(timeindex=date_time_index)
+
+# Read data file
+filename = os.path.join(os.path.dirname(__file__), 'basic_example.csv')
+data = pd.read_csv(filename)
+
+##########################################################################
+# Create oemof object
+##########################################################################
+
+logging.info('Create oemof objects')
+# create natural gas bus
+bgas = solph.Bus(label="natural_gas")
+
+# create electricity bus
+bel = solph.Bus(label="electricity")
+
+energysystem.add(bgas, bel)
```

Comment:
I would suggest a comment like "after creating the bus instances they have to be added to the energysystem manually", 
because I found it a bit counterintuitive that they are not added automatically